### DISCUSSION TITLE
Finally upgraded Mathify - Math Editor to MAUI

### DISCUSSION THREAD

**Discussion by hflexgrig** [dotnet/maui#24013] [Upgrading from Xamarin]
https://play.google.com/store/apps/details?id=com.companyname.mathify

Google has removed my app for 4 months already, because I haven't updated it for 2 years, and the reason of that I couldn't. I have Apple M1 machine, and had multiple failed attempts of migrating to MAUI for those 2 years, and only past week I succeeded, and Google received my changes. The only thing I'll complain is IDE's. I had to use both - VS Code and Jetbrains Rider together, because each of them good on certain things. For example adding new ContentPage or ContentView from vs codes command pallete not working, in the other hand Rider is not good on platform specific codes, and not really recognizes them. And why

**Comment by davidortinau** [comment]
Congrats!

Re: the VSCode templates not working, are they working from the Solution Explorer? It should launch this experience:

<img width="619" alt="Screenshot 2024-08-05 at 9 34 08 AM" src="https://github.com/user-attachments/assets/1a10bd83-2d91-4b7c-ae49-c84860910edc">


Please file your feedback and issues so the C# DevKit and MAUI DevKit teams can see them.

<img width="589" alt="Screenshot 2024-08-05 at 9 37 16 AM" src="https://github.com/user-attachments/assets/36cd510d-71dc-4113-bfbe-0e3f59041e13">

**Reply by hflexgrig** [reply]
> Re: the VSCode templates not working, are they working from the Solution Explorer? It should launch this experience:

Yeah, from solution explorer. Created issue https://github.com/microsoft/vscode/issues/224874
